action.yml

```diff
@@ -12,9 +12,9 @@ inputs:
   jira-project-key:
     description: 'Jira project key where issues will be created'
     required: true
-  jira-host:
-    description: 'Host of the target Jira instance'
-    required: true
+  jira-base-uri:
```

Comment:
I know we said we were going to plan to version the action, so I think we'll need to ensure we're doing that correctly before we merge this PR. Because the update to the `jira-host` input will break this action for anyone currently using it.

Reply:
I changed the target branch to a branch "v2" So we can have multiple PRs targeting that branch until all refactor tasks are complete then we can move v2 over to main once we are ready.